fastlane/spec/actions_specs/ensure_git_status_clean_spec.rb

```diff
@@ -39,6 +39,28 @@
             end").runner.execute(:test)
           end
 
+          it "outputs success message when a file path contains spaces" do
+            allow(Fastlane::Actions).to receive(:sh).with("git status --porcelain", log: false).and_return(" M fastlane/spec/fixtures/git_commit/A FILE WITH SPACE")
```

Comment:
aren't file names with spaces double quoted in `git status` output?

From the manual

"The fields (including the ->) are separated from each other by a single space. If a filename contains whitespace or other nonprintable characters, that field will be quoted in the manner of a C string literal: surrounded by ASCII double quote (34) characters, and with interior special characters backslash-escaped."

Reply:
Right 👍 

```
R  "File with spaces.swift" -> "File with spaces 1.swift"
```

Fixed ✔️